client/verta/verta/tracking/entities/_entity.py

```diff
@@ -126,7 +126,7 @@ def _get_proto_by_id(cls, conn, id):
     @classmethod
```

Comment:
This file's five-line diff is the entirety of the real code change—everything else is pruning unnecessary uses of `_generate_default_name()` out of the test suite.